pkg/asset/machines/machineconfig/hyperthreading.go

```diff
@@ -41,7 +22,8 @@ func ForHyperthreadingDisabled(role string) (*mcfgv1.MachineConfig, error) {
 			},
 		},
 		Spec: mcfgv1.MachineConfigSpec{
-			Config: rawExt,
```

Comment:
https://github.com/openshift/machine-config-operator/blob/master/install/0000_80_machine-config-operator_01_machineconfig.crd.yaml#L70

That makes it feel like ignition field is required

Reply:
Hmm possibly, I didn't test this locally; we also don't have CI coverage for this which probably needs to happen as a prerequisite.   (It is IMO kind of a bug if the ignition section is really required becuase in exactly this case we're just using the "high level" MC section and not Ignition)

Reply:
As of today, at least this has to be passed:
```
ignConfig := igntypes.Config{
    Ignition: igntypes.Ignition{
	Version: igntypes.MaxVersion.String(),
    },
}
```